packages/angular_devkit/core/src/virtual-fs/host/memory.ts

```diff
@@ -211,6 +212,20 @@ export class SimpleMemoryHost implements Host<{}> {
       const content = this._cache.get(from);
       if (content) {
         this._cache.delete(from);
```

Comment:
This side effect should be moved to the end of the function (next to the set) to avoid inconsistent state if an exception is thrown.

Reply:
Done.